src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+
+  div.classList.add('notification');
+  div.classList.add(type);
```

Comment:
```suggestion
  div.classList.add('notification', 'type');
```
I believe that it's possible to do it in this way